presto-main/src/main/java/io/prestosql/sql/planner/LocalExecutionPlanner.java

```diff
@@ -1584,7 +1584,7 @@ public PhysicalOperation visitIndexJoin(IndexJoinNode node, LocalExecutionPlanCo
             }
 
             OperatorFactory lookupJoinOperatorFactory;
-            OptionalInt totalOperatorsCount = getJoinOperatorsCountForSpill(context, session);
```

Comment:
Was this a mistake in the existing code?  I don't think index join needs or supports spilling anyway.

Reply:
Good catch. @sopel39 please extract this change to a separate commit.